api/src/main/java/org/pmiops/workbench/firecloud/FireCloudConfig.java

```diff
@@ -89,4 +89,14 @@ public GroupsApi groupsApi(@Qualifier(ALL_OF_US_API_CLIENT) ApiClient apiClient)
     api.setApiClient(apiClient);
```

Comment:
I'd thought there was a problem with having a "default" and a qualified version of the same class. In my case I had to qualify both.

I might be wrong on that, but I'd double check this is working.

Reply:
Explicitly named both with constants.